src/components/MultiHopTrade/hooks/useIsTradingActive.tsx

```diff
@@ -15,35 +18,78 @@ export const useIsTradingActive = () => {
 
   const swapperName = useAppSelector(selectActiveSwapperName)
 
-  const { data: isTradingActiveOnSellPool } = useQuery({
-    ...reactQueries.common.isTradingActive({
-      assetId: sellAssetId,
-      swapperName,
-    }),
+  const { data: sellAssetInboundAddressData, isLoading: isSellAssetInboundAddressLoading } =
+    useQuery({
+      ...reactQueries.thornode.inboundAddresses(),
+      // @lukemorales/query-key-factory only returns queryFn and queryKey - all others will be ignored in the returned object
+      // Go stale instantly
+      staleTime: 0,
+      // Never store queries in cache since we always want fresh data
+      gcTime: 0,
+      refetchOnWindowFocus: true,
+      refetchOnMount: true,
+      refetchInterval: 60_000,
+      select: data => selectInboundAddressData(data, sellAssetId),
+      enabled: Boolean(sellAssetId),
```

Comment:
will wait for your response RE this pattern in upstream PRs, though I suspect this is the right way of doing things despite being very repeat

Reply:
If we can find some good naming for spreading common args, this PR would definitely be the guy for it 🙏🏽

Reply:
@woodenfurniture resolving as we've discussed this one out and there is not so much of a pattern to abstract things here